st2tests/tests/actions/render.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: render
```

Comment:
Can you name this better so we know what we are testing and what we are rendering?